docs/index.md

```diff
@@ -4,7 +4,141 @@ hide:
   - toc
 ---
 
-<h1 align="center" style="margin-top: 0px;">Dasharo OSF vPub Summer 2022</h1>
-<h2 align="center" style="margin-top: 0px;">More info coming soon!</h2>
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
 
-If you are interested in previous events please visit our [archive](archive.md).
+<p align="center" style="margin-bottom: 0px !important;">
+  <img width="800" src="../images/hackathon-2022-poster.png" alt="Hackathon 2022 poster" align="center">
+  </p>
+<br>
+
+---
+
+The main theme of this year's hackathon will be coreboot. coreboot is an
+extended firmware platform that provides instant and secure booting of
+modern computers and embedded systems. As an open source project, it provides
+auditability and maximum control over the technology.
+
+As an active contributor and promoter of open-source firmware, the 3mdeb Team
+wants to familiarize participants with coreboot. You can also participate in
+providing support for the selected platform (more information in the near
+future).
+
+3mdeb will provide the hardware needed to participate in the event. Bring
+your own hardware as well. If we find that it is already supported in the
+coreboot tree, we will be happy to flash coreboot on your device.
+During the event there will be soft drinks and beer, as well as small snacks.
```

Comment:
Whole description should be about our participants, explaining what is coreboot is ok, but advertising 3mdeb should be at the end not as second paragraph.

Second paragraph should cover content of what you mentioned in third paragraph, but improved. Focus on our participants and what we can provide for them.
- you will learn how to flash coreboot on various hardware platforms using DYI and professional tools
- we will analyze your hardware and feasibility of coreboot flashing, don't worry we can recover if anything will go wrong

We should talk about soft drinks, beer and other stuff in venue section.

Reply:
I changed this description.

Reply:
@Asiderr it is better, but far from perfect. I would appreciate @miczyg1 hand here. @miczyg1 if you could try to improve this section to make it attractive to you. Right now we little bit suggesting things may go wrong, so maybe we should use some slogans related to having fun and fact that there is no lesson without bricking something.

Reply:
@miczyg1 this is damn good. @Asiderr please check what @miczyg1 wrote, this is marketing intro that is needed for such event. Very enthusiastic.